_test/test_sqw_file/bug.m

```diff
@@ -0,0 +1,30 @@
+data = 'w2_small_v1.sqw';
```

Comment:
Debugging file?

Reply:
this one should be removed -- it went into repo erroneously